maintenance/tracing/tracing.go

```diff
@@ -81,11 +69,11 @@ func (h *traceHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // Handler generates a tracing handler that decodes the current trace from the wire.
 // The tracing handler will not start traces for the list of ignoredPrefixes.
-func Handler(ignoredPrefixes ...string) func(http.Handler) http.Handler {
+func Handler() func(http.Handler) http.Handler {
```

Comment:
Should be backwards compatible. I think it would be ok to just revert this file and leave it as it is, as it was working already.